src/MICore/Checksum.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MICore
+{
+    public class Checksum
+    {
+#if CORECLR
```

Comment:
I wonder if we want to just use a string. Its not like MIEngine does any actually hashing. Then we can get rid of all the ifdefs.

---
In reply to: [64833308](https://github.com/Microsoft/MIEngine/pull/346#discussion_r64833308) [](ancestors = 64833308)